massa-execution-worker/Cargo.toml

```diff
@@ -34,6 +34,7 @@ massa_ledger_exports = { path = "../massa-ledger-exports" }
 massa_pos_worker = { path = "../massa-pos-worker", optional = true }
 massa_pos_exports = { path = "../massa-pos-exports" }
 massa_final_state = { path = "../massa-final-state" }
+massa_versioning_worker = { path = "../massa-versioning-worker" }
```

Comment:
I think it's time to rename this module to massa-versioning, because it's not in it's own thread anymore.

Reply:
I'm in favor of doing it in another PR (next one?) what do you think?

Reply:
Sure, it's certainly not breaking for this one!